src/TinyIoC/TinyIoC.cs

```diff
@@ -4246,7 +4243,7 @@ public static Assembly Assembly(this Type type)
     }
 #endif
     // reverse shim for WinRT SR changes...
-#if (!NETFX_CORE && !PORTABLE && !NETSTANDARD1_0 && !NETSTANDARD1_1 && !NETSTANDARD1_2 && !NETSTANDARD1_3 && !NETSTANDARD1_4 && !NETSTANDARD1_5 || !NETSTANDARD1_6)

```

Comment:
Yes, net standard 1.6 may support this, we didn't know at the time. As new runtimes come out this will have to be updated / simplified